### PR TITLE
Update debugging.md

### PR DESCRIPTION
**Pull request by locks**
I don't think `Ember.TEMPLATES` is populated anymore.

### CODE REVIEW

**Comment by jenweber** [comment]
@locks can you confirm this by testing in an app or checking the Ember codebase?

File: guides/v3.6.0/configuring-ember/debugging.md [hunk 1]
```diff
@@ -19,29 +19,18 @@ export default Application.extend({
...
-## Views / Templates
-
-#### Log view lookups
-
-```javascript {data-filename=config/environment.js}
-ENV.APP.LOG_VIEW_LOOKUPS = true;
-```
-#### View all registered templates
-```javascript
-Ember.keys(Ember.TEMPLATES)
```
Comment (locks) [inline guides/v3.6.0/configuring-ember/debugging.md]:
```
$ Ember.keys(Ember.TEMPLATES)
TypeError: Ember.keys is not a function[Learn More]
```

File: guides/release/configuring-ember/debugging.md [hunk 2]
```diff
@@ -21,41 +21,41 @@ export default Application.extend({
...
 ## Views / Templates
 
-#### Log view lookups
+### Log view lookups
 
 ```javascript {data-filename=config/environment.js}
 ENV.APP.LOG_VIEW_LOOKUPS = true;
 ```
-#### View all registered templates
+### View all registered templates
```
Comment (rwjblue) [inline guides/release/configuring-ember/debugging.md]: This isn’t a thing anymore

File: guides/release/configuring-ember/debugging.md [hunk 3]
```diff
@@ -70,15 +70,15 @@ export default Application.extend({
   LOG_RESOLVER: true
 });
 ```
-#### Dealing with deprecations
+### Dealing with deprecations
```
Comment (rwjblue) [inline guides/release/configuring-ember/debugging.md]: We should talk about ember-cli-deprecation-workflow  it these flags

File: guides/release/configuring-ember/debugging.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ Ember = {
 };
 ```
 
-#### Errors within an `RSVP.Promise`
+### Errors within an `RSVP.Promise`
```
Comment (rwjblue) [inline guides/release/configuring-ember/debugging.md]: I think this is “fine”, but _in general_ folks shouldn’t need this anymore.

File: guides/release/configuring-ember/debugging.md (hunk 3 above)
Comment (locks) [reply guides/release/configuring-ember/debugging.md]: https://github.com/ember-learn/guides-source/issues/475

File: guides/release/configuring-ember/debugging.md (hunk 4 above)
Comment (locks) [reply guides/release/configuring-ember/debugging.md]: Removed for simplicity!

File: guides/v3.6.0/configuring-ember/debugging.md (hunk 1 above)
Comment (mike-north) [reply guides/v3.6.0/configuring-ember/debugging.md]: > I don't think Ember.TEMPLATES is populated anymore.

Confirmed in latest canary, `Ember.TEMPLATES` is always an empty object

**Comment by acorncom** [comment]
Netlify deploys failed, but the Travis / Percy tests look fine

**Comment by locks** [comment]
@acorncom you have a netlify tied to this repo and that's failing because it uses node 6 :x The Ember team one passes.

**Pull request merged**